plugin/src/test/groovy/com/github/triplet/gradle/play/PlayPublishTaskTest.groovy

```diff
@@ -111,6 +112,34 @@ class PlayPublishTaskTest {
         doReturn(editCommitMock).when(editsMock).commit(anyString(), anyString())
     }
 
+    @Test
+    void whenPublishing_withOverrideFolder_UploadsOnlySpecifiedFiles() {
```

Comment:
Yeah, Gradle doesn't support testing incremental inputs very well. I personally don't see the point of testing a very simple piece of code so I say we remove the tests so you can focus on the live testing stuff which will be way more useful.